src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const STATUS = 'STATUS';
```

Comment:
- [ ] **Required:** You have implemented the required changes by the previous reviewer but missed this one. Kindly implement it as well.

Reply:
Thanks, @codepantha, the requirement was implemented.